src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -534,17 +534,13 @@ ThreadDumpDCmd::ThreadDumpDCmd(outputStream* output, bool heap) :
 }
 
 void ThreadDumpDCmd::execute(DCmdSource source, TRAPS) {
-  // thread stacks
-  VM_PrintThreads op1(output(), _locks.value(), _extended.value());
+  // thread stacks and JNI global handles
+  VM_PrintThreads op1(output(), _locks.value(), _extended.value(), true);
```

Comment:
Please comment what the true argument means.

Reply:
fixed